src/main/scala/fi/oph/koski/validation/KoskiValidator.scala

```diff
@@ -1567,4 +1568,14 @@ class KoskiValidator(
       case _ => HttpStatus.ok
     }
   }
+
+  private def validateSuorituskieli(suoritus: Suoritus) = {
+    def validateSuomiTaiRuotsi(koodiarvo: String) = HttpStatus.validate(List("SV", "FI").contains(koodiarvo)) {
+      KoskiErrorCategory.badRequest.validation.rakenne.virheellinenSuorituskieli("Suorituskielen tulee olla suomi tai ruotsi")
+    }
+    suoritus match {
+      case k: PerusopetukseenValmistavanOpetuksenSuoritus => validateSuomiTaiRuotsi(k.suorituskieli.koodiarvo)
```

Comment:
jos/kun tätä on tarkoitus laittaa moneen eri tyyppiseen suoritukseen: olisikohan siistimpää, jos traitista Suorituskielellinen perisi validaation tarvitseman traitin (esim. VainSuomiRuotsiSuorituskielellinen jne.), ja sitten tässä validaatiossa ja tarvittavissa päätason suoritus-luokissa käyttäisi näitä traitteja?

Silloin ehkä vähemmän helposti unohtuisi lisätä mahdolliset uudet validaatiota tarvitsevat suoritustyypit tänne, kun niitä luodaan.